Gemfile

```diff
@@ -11,7 +11,7 @@ gem "hoe-bundler", "~>1.2", :group => [:development, :test]
 gem "hoe-git", "~>1.6", :group => [:development, :test]
 gem "minitest", "~>5.10.1", :group => [:development, :test]
 gem "mocha", "~>1.5.0", :group => [:development, :test]
-gem "rubocop", "~>0.56.0", :group => [:development, :test]
+gem "rubocop", "~>0.51.0", :group => [:development, :test]
```

Comment:
I'll add a comment reminding us, that we use this specific version of rubocop for codacy (otherwise we might just update it in the future)

Reply:
Actually we need to change it in the Rakefile and regenerate the Gemfile....